src/cbmc/cbmc_parse_options.cpp

```diff
@@ -75,6 +76,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <goto-instrument/nondet_static.h>
 #include <goto-instrument/reachability_slicer.h>
 
+#include <goto-instrument/abstraction_spect.h>
```

Comment:
Check the coding standard but there was an attempt to have the headers sorted alphabetically.  Certainly with the other `goto-instrument` includes would be good.

Reply:
We have decided to have --use-rra option only for goto-instrument, not for cbmc. This code disappears in the next commit.

Reply:
Then please remove it from the PR.  Ideally each PR we merge should be a finished thing and not depend on others.